draft-rfcs/resources/pre-rfc-notes.md

```diff
@@ -0,0 +1,28 @@
+# Pre-RFC process
+
+## Motivations and Reasons
+* navigating the RFC process can be intimidating
+* makes it more likely for the eventual RFC to be accepted
+* we often get RFCs "out of the blue"
+* it would be better to clarify the ideas that are "moving forward" ideas that are "speculative"
+* no sense in spending the time writing a full, formal RFC without the idea that somebody on the team is into the idea
+
+
+## Process
+* Start with an open issue and an internals thread
+* If someone is interested in idea or team wants to pick it up, "liason" gets involved in the discussion and works with the people pushing the idea
```

Comment:
How is liaison designated , someone fulfilling the requirements (maybe a link here?) just joins the conversation in the thread?